packages/odyssey-storybook/src/components/odyssey-mui/Status/Status.stories.tsx

```diff
@@ -74,6 +79,19 @@ const storybookMeta: Meta<StatusProps> = {
 
 export default storybookMeta;
 
+const checkStatusStyles = async (
+  canvas: ReturnType<typeof within>,
+  statusLabel: string,
+  expectedBackgroundColor: string,
+  expectedColor: string,
+) => {
+  const status = canvas.getByText(statusLabel);
+  await expect(status).toHaveStyle(
+    `background-color: ${expectedBackgroundColor}`,
+  );
+  await expect(status).toHaveStyle(`color: ${expectedColor}`);
+};
```

Comment:
Can we use Test Selectors here so we start putting that code in use?

Reply:
Sure, I updated the tests to include an ID e.g. `testId: "button-medium",`  (I believe that's what we're after, right?)